templates/network/network.py

```diff
@@ -114,16 +118,11 @@ def configure(self):
             ovs_container_name='ovs',
             bonded=self.data.get('bonded', False),
             mtu=self.data.get('mtu', 9000) or 9000,
+            mode=self.data.get('mode', 'ovs'),
         )
 
         self.state.set('actions', 'install', 'ok')
 
     def uninstall(self):
-        try:
-            container = self._node_sal.containers.get('ovs')
-            self._node_sal.network.unconfigure('ovs')
-            container.stop()
-        except LookupError:
-            pass
-
+        self._node_sal.network.unconfigure('ovs', self.data.get('mode', 'ovs'))
```

Comment:
not more stopping of the ovs container ?

Reply:
It's done in the sal, since the sal is actually what creates the ovs container